packages/cli/src/routeGeneration/templates/koa.hbs

```diff
@@ -92,7 +92,7 @@ export function RegisterRoutes(router: KoaRouter) {
             } catch (err) {
               const error = err as any;
               context.status = error.status;
-              context.throw(error.status, JSON.stringify({ fields: error.fields }));
+              context.throw(context.status, error.message, error);
```

Comment:
I think we can just rethrow, right?
And set expose to true:
https://github.com/koajs/koa/blob/master/docs/api/context.md#ctxthrowstatus-msg-properties